### ISSUE TITLE
[BUG] Missing achievements 

### ISSUE THREAD

**Issue by ZimGil** [Kibibit/achievibit#196]
So there are 2 achievements that I should have got which I didn't

 - The first was `Don't Yell at Me` (#60) that I should have received from merging Kibibit/achievibit#189
     > Unit tested and approved that achievement should have been granted.

 - The second was `Cutting Edges` (#171) that I should have received for merging ZimGil/ngWiz#88

**Comment by Thatkookooguy** [comment]
@ZimGil those are two different potential bugs :-)

You shouldn't have got `Cutting Edge` on ZimGil/ngWiz#88 because that achievement is not in production yet. It's merged to the `development` branch but wasn't merged into the `master` branch yet.
Currently, the production branch of achievibit is being deployed from `develop` manually and that didn't happen since `Cutting Edges` was merged.

For `Don't Yell at Me`, that's **a real bug**!
The entire system will be replaced in version 2, which will probably solve this error so I wouldn't waste too much time on fixing version 1. But if you want to investigate this and fix it in this version, go ahead!
Basically the achievements will pass **as is** to version 2, but the system that grants them and collects the data is going to get a makeover

I'll change this into a generic issue which will be fixed after the base of version 2 will be deployed
